xdsl/frontend/onnx/ir_builder.py

```diff
@@ -0,0 +1,25 @@
+from onnx import ValueInfoProto
+
+from xdsl.frontend.onnx.type import get_type
+from xdsl.ir import Attribute
+
+
+class OnnxXdslMapping:
+    """The representation of the onnx context."""
```

Comment:
Yeah, but what does it do? Why is it useful? A better docstring would be appreciated